LocalWiki.php

```diff
@@ -249,6 +249,34 @@ function onBeforePageDisplay( OutputPage $out ) {
 			$out->addLink( [ 'rel' => 'stylesheet', 'href' => 'https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap' ] );
 		}
 
+		break;
+	case 'paneidoversewiki':
+		$wgHooks['AdminLinks'] = 'onAdminLinks';
+		function onAdminLinks( &$adminLinksTree ) {
+			$general = $adminLinksTree->getSection( wfMessage( 'adminlinks_general' )->text() );
+			$generalRow = $general->getRow( 'main' );
+			$generalRow->addItem( ALItem::newFromSpecialPage( 'TimeMachine' ) );
+			$generalRow->addItem( ALItem::newFromSpecialPage( 'ArticlesHome' ) );
+			$generalRow->addItem( ALItem::newFromSpecialPage( 'EditWatchlist' ) );
+			$generalRow->addItem( ALItem::newFromSpecialPage( 'GlobalPreferences' ) );
+			$generalRow->addItem( ALItem::newFromSpecialPage( 'UploadFile' ) );
+			$generalRow->addItem( ALItem::newFromEditLink( 'Portal', 'Draft portal' ) );
+			$generalRow->addItem( ALItem::newFromEditLink( 'Common.js', 'Edit JS file' ) );
+
+			$cosmos = $adminLinksTree->getSection( wfMessage( 'skinname-cosmos' )->text() );
+			$cosmosRow = $cosmos->getRow( 'cosmos' );
+			$cosmosRow->addItem( ALItem::newFromEditLink( 'Cosmos.js', 'Edit skin JS' ) );
```

Comment:
Cosmos already adds these links to AdminLinks I believe.

Reply:
I don't see those links at https://readerswhoknow.paneidoverse.com/wiki/Special:AdminLinks, nor did I see them when looking at Cosmos' AdminLinks hook.

Reply:
Interesting. I guess they aren't. Maybe I only thought about adding them to the hook.

Reply:
https://gerrit.wikimedia.org/r/c/mediawiki/skins/Cosmos/+/884012

Reply:
I'll remove the cosmos links then.